pkg/lwip/Makefile.include

```diff
@@ -17,6 +17,12 @@ PSEUDOMODULES += lwip_udp
 PSEUDOMODULES += lwip_udplite
 PSEUDOMODULES += lwip_sock_async
 
+#These flags enable calculating ICMP chksum in LWIP software.
+#Without these flags RIOT device respond with ICMP chsum equal to 0, and
+#cannot be successfully "pinged" from Windows hosts.
+CFLAGS +=-DCHECKSUM_GEN_ICMP=0
+CFLAGS +=-DLWIP_CHECKSUM_CTRL_PER_NETIF=1
```

Comment:
This would add runtime overhead so that checksum generation and validation can be enabled/disabled for each network interface. IMO it would be better to just enable ICMP checksum generation for all interface at compile time by default. If there really is need to fine-tune checksum generation and validation, this could be exposed as a module that users would have to enable explicitly in order to trade some RAM and CPU cycles for the ability to fine-tune checksum settings at runtime.

Reply:
I don't understand idea of special module you mentioned. Could you send link to similar module which is currently in RIOT?

After @yarrick  tests I suspect that this issue concerns only subset of boards - probably Nucleo boards. Next week I should heva access to other Nucleo boards - I perform additional experiments. But now, I have other solution - maybe conditional addition of these flags - only for required boards is simpler solution than additional module about which programmer must remember.